### DISCUSSION TITLE
Common Q&A

### DISCUSSION THREAD

**Discussion by arbaabgtamods** [sannybuilder/dev#332] [Q&A] [answered]
I want a new future in sanny builder 3 plzz add in opcode car headlights toggle for gta vice city please add !!

**Comment by MiranDMC** [comment, marked as answer]
> I want a new future in sanny builder 3 plzz add in opcode car headlights toggle for gta vice city please add !!

This possibly can be done with memory related opcodes:
https://github.com/DK22Pac/plugin-sdk/blob/master/plugin_vc/game_vc/CVehicle.h#L125

**Reply by arbaabgtamods** [reply]
Hello sir i want sanny builder and vc cleo with my opcode car headlights toggle and tell me what opcode headlights control to gta vc plzz

**Comment by arbaabgtamods** [comment]
sir i dont know how to use it plzz give me  a simple opcode plzz for car
headlights and more like door lock unlock and🥹

**Comment by arbaabgtamods** [comment]
And Sir Can You Sent Me Sanny builder 3 With In Opcode Car Headlights And
More! plzz i dont know what i do your file (CVehicle.h)
